app/javascript/views/usage/index.js

```diff
@@ -24,33 +24,38 @@ $(() => {
   initializeCharts();
 
   // Create the Users joined chart
-  const usersData = JSON.parse($('#users_joined').val());
-  if (isObject(usersData)) {
-    createChart('#yearly_users', usersData);
+  if (!isUndefined($('#users_joined').val())) {
```

Comment:
good. I hope these checks become necessary in future versions of Rails' asset pipeline. It would be better if it just sent the client browser the JS it actually needs for the page.